packages/generator/CHANGELOG.md

```diff
@@ -1,5 +1,85 @@
 ## CHANGELOG
 
+### v1.3.0-alpha.8
+
+### **Core Functionality**
+
+#### Authentication helper changes
+
+* **BREAKING**: helpers.getAuthToken method is replaced by helpers.getShopperToken
+* helpers.getShopperToken supports Client Configuration. Refer to [Sample Code](./README.md#Sample Code)
+
+#### API Clients
+
+* **BREAKING**: API client MUST be instantiated using API name, e.g. `new Product.ShopperProduct({})`
```

Comment:
```suggestion
* **BREAKING**: API client MUST be instantiated using API name, e.g. `new Product.ShopperProduct(clientConfig)`
```

Thinking we should reiterate using something like "clientConfig" when showing example code"